### PR TITLE
minor bug fix, allows for periods to be Listed in SwiftUI

### PR DESCRIPTION
**Pull request by xt0fer**
adding a id field and implementing Identifiable, allows the items in the Period to listed in a SwiftUI List or a ForEach iterator. Also fixed a minor bug where latitude was being sent instead of longitude,

### CODE REVIEW

**Comment by xt0fer** [comment]
to be more clear, the id field and Identifiable adherence in the Forecast.Period lets you do something like
```
struct FutureView: View {
    var forecast: Forecast
    
    // For formatting the forecast periods.
    var body: some View {
        ForEach(forecast.periods) { period in
            VStack(alignment: .leading){
                Text("\(period.name!)")
                    .font(.title2)
                Text("\(period.detailedForecast!)")
                    .font(.body)
                HStack{
                    Text("Temp: \(period.temperature.formatted())")
                    Text("Wind: \(period.windSpeed.description)")
                }
                .font(.body.bold())
            }
            .padding(.bottom, 1)
        }
    }
}
```

**Comment by ualch9** [comment]
I will review this hopefully sometime soon, thank you for the PR!

File: Sources/NationalWeatherService/services/calls/NWS+Forecast.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ extension NationalWeatherService {
     fileprivate func loadForecast(at url: URL, then handler: @escaping ForecastHandler) {
         var components = URLComponents(url: url, resolvingAgainstBaseURL: false)!
         components.queryItems = [
-            URLQueryItem(name: "units", value: "si")
```
Comment (ualch9) [inline Sources/NationalWeatherService/services/calls/NWS+Forecast.swift]: why was this removed?

File: Sources/NationalWeatherService/services/calls/NWS+CurrentCondition.swift [hunk 2]
```diff
@@ -39,7 +39,7 @@ import CoreLocation
 extension NationalWeatherService {
     /// Gets the current weather for the given location.
     public func currentCondition(for coordinates: CLLocationCoordinate2D, then handler: @escaping CurrentConditionHandler) {
-        self.currentCondition(latitude: coordinates.latitude, longitude: coordinates.latitude, then: handler)
```
Comment (ualch9) [inline Sources/NationalWeatherService/services/calls/NWS+CurrentCondition.swift:42]: 👍

File: Sources/NationalWeatherService/models/Forecast/ForecastPeriod.swift [hunk 3]
```diff
@@ -8,13 +8,14 @@
...
 extension Forecast {
-    public struct Period: Decodable {
+    public struct Period: Decodable, Identifiable {
         public enum CodingKeys: String, CodingKey {
             case name, startTime, endTime, isDaytime
             case temperature, temperatureUnit, windSpeed, windDirection
             case icon, shortForecast, detailedForecast
         }
 
+        public let id = UUID()
```
Comment (ualch9) [inline Sources/NationalWeatherService/models/Forecast/ForecastPeriod.swift:18]: 👍

**Review by ualch9 — CHANGES_REQUESTED** [review]
Thanks for your PR. If you could just clarify one comment, I would greatly appreciate it!

File: Sources/NationalWeatherService/services/calls/NWS+Forecast.swift (hunk 1 above)
Comment (xt0fer) [reply Sources/NationalWeatherService/services/calls/NWS+Forecast.swift]: I did not need the results to be in metric units. I can remove it from the PR if you'd like. The best thing might be to have some kind of config mech for it, but I haven't thought that hard on it yet.

Comment (ualch9) [reply Sources/NationalWeatherService/services/calls/NWS+Forecast.swift]: Since the library is providing temperature as a [Measurement](https://github.com/WeatherProvider/NationalWeatherService-Swift/blob/562a235eb991d78ad07b0dd473b4e147f889c469/Sources/NationalWeatherService/models/Forecast/ForecastPeriod.swift#L23), the programmer can convert between customary/SI as needed [[Apple Docs](https://developer.apple.com/documentation/foundation/measurement/2799413-converted)]. Just to make sure that NWS is returning SI, I would rather keep the `units=si` in the URL. If you have any other feedback, we can continue to discuss, otherwise let's remove this change.

Comment (xt0fer) [reply Sources/NationalWeatherService/services/calls/NWS+Forecast.swift]: restored to orig

**Comment by xt0fer** [comment]
restored to orig metric unit request

**Comment by xt0fer** [comment]
wait, I should not have closed the PR.

**Comment by ualch9** [comment]
CI is failing because Actions' secrets are not given to PRs. Tested on my own machine (simulator & macOS) and tests pass.

**Comment by ualch9** [comment]
thank you very much, @xt0fer, for your pr. I really appreciate it!!
